README.md

```diff
@@ -1 +1,142 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowlednpments](#acknowlednpments)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 school-library<a name="about-project"></a>
+
+**school-library** This project is a school Library app built using Ruby which will allow users to Add new students or teachers, Add new books and Save records of who borrowed a given book and when. It is an app to record what books are in the library and who borrows them.
+
+## 🛠 Built With <a name="built-with"></a>
+RUBY
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Key Features
+
+- Person, Student and Teacher classes has been implemented
+- Will add more classes and functionalionaties in the upcoming milestones of this project
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+  ruby > 3.2.2
+  code-runner
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:baqar-abbas/school-library.git
+```
+
+### Usage
+
+To run the project, execute the ruby file with any code runner of your choice.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="Shaheryar-abid"></a>
+
+👤 **Shaheryar-abid**
+
+- GitHub: [@githubhandle](https://github.com/Shaheryar0054)
+- Twitter: [@twitterhandle](https://twitter.com/sharyar0310)
+- LinkedIn: [@LinkedIn](https://www.linkedin.com/in/shaheryar-abid/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- More classes and functionalities will be added in the future milestones of this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project please follow and give it a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowlednpments"></a>
+
+> I would like to Thanks Microverse for the idea of this Program
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ <a name="faq"></a>
+
+> Is this an Open Source Project?
+
+Yes
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```

Comment:
| @Shaheryar0054 REPO MIT Error |
|-|
| ![image](https://github.com/Shaheryar0054/School-library/assets/93039359/8c8e0c77-0c40-4963-8ccb-3bac46e824bb) |

- [ ] It would be nice if you fix this MIT link because it is broken 😨 Kindly fix it to make your readme free from broken links which makes the readme not professional 🤔

**Hint: You don't have any LICENSE files in your repo 🤔 Kindly add a LICENSE file using these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 👍**